### PR TITLE
feat(core): use new ETH send flow also for staking

### PR DESCRIPTION
**Pull request by ibz**

### CODE REVIEW

File: core/src/trezor/ui/layouts/mercury/__init__.py
Comment (obrusvit) [inline core/src/trezor/ui/layouts/mercury/__init__.py:1091]: Why is `chunkify` set to `False`?

File: core/src/trezor/ui/layouts/mercury/__init__.py
Comment (obrusvit) [inline core/src/trezor/ui/layouts/mercury/__init__.py:1104]: What is the comment about?

**Review by obrusvit — COMMENTED** [review]
Tests with input flows for ETH signtx and staking are still skipped. Otherwise, it looks good. We should take a look at the parameter set of the flows, calling a function with 18 arguments does not seem good.

File: core/src/trezor/ui/layouts/mercury/__init__.py
Comment (obrusvit) [reply core/src/trezor/ui/layouts/mercury/__init__.py:1091]: Ok, I wasn't aware that we *always* chunkify in `mercury`.

File: core/src/trezor/ui/layouts/mercury/__init__.py
Comment (ibz) [reply core/src/trezor/ui/layouts/mercury/__init__.py:1104]: The title of the Cancel screen, according to Figma, should be "Cancel staking" not "Cancel" like the button, so we need to pass 2 separate parameters here, but I got stuck in trying to make the Rust compiler happy with the types, so I went for the compromise of just calling it "Cancel" all over.

**Comment by ibz** [comment]
> We should take a look at the parameter set of the flows, calling a function with 18 arguments does not seem good.

I know, this flow grew too complex suddenly... It'll soon also make coffee for you after you click "Sign". I need to break it down into separate "versions".

File: core/src/trezor/ui/layouts/mercury/__init__.py
Comment (ibz) [reply core/src/trezor/ui/layouts/mercury/__init__.py:1091]: > Ok, I wasn't aware that we *always* chunkify in `mercury`.

I mean, we always chunkify the address on the (separate) address screen in mercury, because this is the only place we have that screen (it is the one I added now in this PR, so it is only here).

**Review by obrusvit — APPROVED** [review]
Please wait for CI - update `fixtures.json`
Run `make style`.

**Comment by Hannsek** [comment]
I cannot see the diffs :/

**Comment by ibz** [comment]
> I cannot see the diffs :/

They should be visible now. They were not before, because there were no tests enabled. After @obrusvit fixed the input flows and enabled the relevant tests, the screens appeared. But they are not really a diff, because they are new screens, so there is nothing to compare to...

**Comment by Hannsek** [comment]
New screens? We had staking before. Sorry, but I am confused now. @obrusvit can you clarify please?

**Comment by obrusvit** [comment]
> New screens? We had staking before. Sorry, but I am confused now. @obrusvit can you clarify please?

We didn't have UI tests for staking on TS5 since TS5 release until now. 
We had functional tests for the actual signing of the staking operations but the UI flow was tested manually only.

This PR enables the UI tests of staking for TS5 and implements them properly. That's why the screens are new.

**Comment by Hannsek** [comment]
Perfect, thanks for clarifying.

**Comment by bosomt** [comment]
**QA OK**

 - Device: Trezor T3T1 2.8.2 regular (revision d7e1f0faf3ab8753dfd9dc292b503dff39fd81db)

**Pull request merged**
